parser/Parser/test_Parser.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+from Parser import Parser
+import re
+
+def test_filename():
+    x = Parser("test")
+    x.file_name()
+    assert x.filename == "test.txt"
+    x = Parser("test.txt")
+    x.file_name()
+    assert x.filename == "test.txt"
+    x = Parser("t")
+    x.file_name()
+    assert x.filename == "t.txt"
+
+def test_fileiter():
+    x=Parser("test")
+    x.file_name()
+    file=x.file_iter()
+    counter=0
+    language_code_prefix = re.compile('[a-zA-Z][a-zA-Z]:')
+    for line in file:
+        counter+=1
+        assert line == '' or line[0]=="#" or 'stopword' in line or 'synonym' in line or line[0]=="<" or language_code_prefix.match(line) or ":" in line
+        if counter==27:
+            assert line == "carbon_footprint_fr_foodges_value:fr:10"
+    assert counter == 37+1
+
+def test_normalizing():
+    x=Parser("")
+    text="Numéro #1, n°1 des ¾ des Français*"
+    text=x.normalizing(text,"fr")
+    assert text == "numero-1-n-1-des-des-francais"
+    text="Randôm Languäge wìth àccénts"
+    text=x.normalizing(text,"fr")
+    assert text == "random-language-with-accent"
+
+def test_harvest():
+    x=Parser("test")
+    x.file_name()
+    data=x.harvest()
+    test_data=["# test taxonomy",
```

Comment:
maybe you can put it in a test.json file that you parse to get test_data (`json.load`) ? It will avoid cluttering the test, and we immediatly see it goes along with test.txt.